sdk/python/foundation-models/system/reinforcement-learning/reinforcement-learning.ipynb

```diff
@@ -325,32 +327,47 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Function which invokes the model evaluation pipeline.\n",
-    "eval_job, status = run_evaluation_pipeline(\n",
-    "    ml_client=ml_client,\n",
-    "    registry_ml_client=registry_ml_client,\n",
-    "    compute_cluster=\"k8s-a100-compute\",\n",
-    "    grpo_model_dir=grpo_registered_model.path,  # Output from GPRO RL provided as data asset created from earlier step.\n",
-    "    rlpp_model_dir=rlpp_registered_model.path,  # Output from Reinforce_plus_plus RL provided as data asset created from earlier step.\n",
-    "    validation_dataset_path=test_data_path,  # Path to test dataset\n",
-    "    run_config={\n",
-    "        \"num_nodes\": 1,  # Number of nodes to be used for evaluation run.\n",
-    "        \"number_of_gpu_to_use\": 8,  # Number of GPUs in a node to be used for evaluation run.\n",
-    "        \"base_path_1_label\": \"GRPO\",  # Label to identify GRPO model outputs.\n",
-    "        \"base_path_2_label\": \"RLPP\",  # Label to identify RLPP model outputs.\n",
-    "        \"explore_pattern_1\": \"global_step_{checkpoint}/actor/lora_adapter/\",\n",
-    "        \"explore_pattern_2\": \"global_step_{checkpoint}/actor/lora_adapter/\",\n",
-    "        \"checkpoint_values_1\": \"12\",\n",
-    "        \"checkpoint_values_2\": \"12\",\n",
-    "        \"use_lora_adapters_1\": True,\n",
-    "        \"use_lora_adapters_2\": True,\n",
-    "        \"evaluate_base_model\": True,  # Set to True to evaluate base model along with RL finetuned models.\n",
-    "        \"hf_model_id\": \"deepseek-ai/DeepSeek-R1-Distill-Qwen-7B\",  # Huggingface ID of the base model\n",
-    "        \"max_prompt_length\": 8196,\n",
-    "        \"max_response_length\": 1024,\n",
-    "        \"dtype\": \"bfloat16\",\n",
-    "        \"tensor_parallel_size\": 4,\n",
-    "    },  # Configuration parameters for evaluation run.\n",
+    "grpo_model_asset_id = get_run_output_assetid(\n",
+    "    ml_client, job_name=grpo_job.name, output_name=\"model_output\"\n",
```

Comment:
seems to be taking older version of PR => https://github.com/Azure/azureml-examples/pull/3709. Could we take a pull here please?

Reply:
updated, thanks!